server/static/a-saturday-night/index.html

```diff
@@ -100,10 +100,22 @@
 
 			</a-assets>
 
-			<a-entity id="player" networked="template:#player-template;showLocalTemplate:false;" spawn-in-circle="radius:2;" wasd-controls show-avatar="index:0;numAvatars:3" randomize-avatar>
-			  <a-entity camera="userHeight: 1.6" look-controls networked="template:#head-template;showLocalTemplate:false;"></a-entity>
-			  <a-entity networked="template:#left-hand-template" hand-components="left" show-in-vr visible="false"></a-entity>
-			  <a-entity networked="template:#right-hand-template" hand-components="right" show-in-vr visible="false"></a-entity>
+			<a-entity id="player" networked="template:#player-template" spawn-in-circle="radius:2;" wasd-controls show-avatar="index:0;numAvatars:3" randomize-avatar>
+			  <a-entity camera="userHeight: 1.6" look-controls networked="template:#head-template"></a-entity>
+			  <a-entity networked="template:#left-hand-template" hand-components="left" show-in-vr visible="false">
```

Comment:
Shouldn't these `networked` tags use `attachLocalTemplate: false;`? Because they are defining the child entities locally. In this case it seems like we can just remove the child entities however because they are the same as the template.